### DISCUSSION TITLE
Where is the documentation?

### DISCUSSION THREAD

**Discussion by nova-nowiz** [rise-lang/shine#99] [Q&A] [answered]
Hi, I would like to know where the documentation is for the language as I would like to see how it looks and what features is has.
I found the doc repository with latex sources, but where are is the generated content of this documentation?

**Comment by michel-steuwer** [comment, marked as answer]
Hi @Narice

The current documentation of the language is not very good. This is an area we are lacking and actively working on improving.

To get a feel for using the Rise language, you can have a look at the tests, particular in the `apps` folder:
https://github.com/rise-lang/shine/tree/master/src/test/scala/apps

To understand the underlying language and its type system, you can have a look at this [PDF document](https://github.com/rise-lang/shine/files/5917299/rise-dpia-language.pdf) that contains a (partial) formal characterisation of the language (using it's predecessors name of Lift instead of Rise).

Please feel free to ask questions here as well.

**Reply by michel-steuwer** [reply]
@Narice we have started to put more structured documentation together at: https://rise-lang.org/doc/

Still a work in progress. But a starting point.

**Reply by nova-nowiz** [reply]
nice awesome!
